Drop Python2 and Six

oz123 opened the pull request.

matteius commented on src/vistir/contextmanagers.py:
```diff
@@ -345,7 +344,8 @@ def open_file(
                 session = Session()
         if session is None:
-            with closing(urlopen(link)) as f:
+            import urllib.request
```
This style of import is problematic for pipenv to rewrite -- I converted `requirementslib` off this style imports for that reason, can you do `from urllib import request`?  Also why not have it at the top of the file?

matteius replied:
may not matter for this though since its a python builtin library.

oz123 replied:
I fixed this as requested.

matteius commented:
Looks like two editor .swp files got added.  Also this would be a good chance to fix the import issues/catching ModuleNotfound error that caused some issues with removing yaspin:  https://github.com/pypa/pipenv/pull/5268

oz123 commented:
> Looks like two editor .swp files got added. Also this would be a good chance to fix the import issues/catching ModuleNotfound error that caused some issues with removing yaspin: [pypa/pipenv#5268](https://github.com/pypa/pipenv/pull/5268)

Removed the `swp` files from vim, also added these to git ignore.

As for the issue with yaspin, I can't reproduce it. However, pipenv does not even use yaspin, and vistir has yaspin only as an optional dependency. Hence, it should be safe to remove them from pipenv.

matteius commented:
> As for the issue with yaspin

@oz123 -- basically because in your PR the module folder itself gets deleted, some pythons will apparently raise `ModuleNotFoundError` instead of `ImportError` in this case, so this line should change to catch both error types and it should become safe for all:  https://github.com/sarugaku/vistir/blob/master/src/vistir/contextmanagers.py#L180
